api/docs/developer-system-initialization.md

```diff
@@ -2,6 +2,31 @@
 This is a guide to some first-time activities to run after installing the
 software in [System Requirements](developer-system-requirements.md).
 
+## Gradle setup
+
+If your system's default `java` version is 8, you can skip this.
+
+```Shell
+java --version
+openjdk 11.0.11 2021-08-03
+...
+```
+
+Gradle requires Java 8 to build currently (RW-5609 will upgrade to 11).
+
+- Ensure Java 8 is installed on your system
+- Create a Gradle properties file to point to this version:
+  - Globally for your workstation: `~/.gradle/gradle.properties`
+  - or, just in the Workbench repo: `api/gradle.properties`
+  ```
+  # gradle.properties
+  org.gradle.java.home={PATH_TO_JDK8}
+
+  # Example:
+  $ cat api/gradle.properties
+  org.gradle.java.home=/usr/local/buildtools/java/jdk8
```

Comment:
I'm happy to add it, but I don't use a Mac and searching around I don't see a clear answer. Do you have a path to suggest?

Reply:
/usr/libexec/java_home -v 1.8

Reply:
ah - yes it's a command line.  The reason this is the preferred method on Mac is because Apple likes to change the actual path with new releases.  Mine is `/Library/Java/JavaVirtualMachines/adoptopenjdk-8-openj9.jdk/Contents/Home` but that's not universal.